packages/eas-cli/src/commands/workflow/validate.ts

```diff
@@ -0,0 +1,83 @@
+import fs from 'fs/promises';
+import yaml from 'js-yaml';
+import path from 'path';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import Log from '../../log';
+import { ora } from '../../ora';
+
+export class WorkflowValidate extends EasCommand {
+  static override description = 'validate a workflow configuration yaml file';
+
+  static override args = [
+    {
+      name: 'path',
+      description: 'Path to the workflow configuration YAML file (must end with .yml or .yaml)',
+      required: true,
+    },
+  ];
+
+  async runAsync(): Promise<void> {
+    const {
+      args: { path: filePath },
+    } = await this.parse(WorkflowValidate);
+
+    const spinner = ora().start('Validating the workflow YAML file…');
```

Comment:
nit: does validating take so long that it requires a spinner?

Reply:
I'm not sure if it's worth complicating things, but in some other places we use a spinner if the action takes > 1s:
https://github.com/expo/eas-cli/blob/d5a9e76fd470b2a941a4ba11656ec091e7066641/packages/eas-cli/src/build/utils/repository.ts#L130-L139